app/src/main/java/com/p2p/meshify/MainActivity.kt

```
@@ -73,26 +66,22 @@ class MainActivity : ComponentActivity() {
             val navController = rememberNavController()
 
             LaunchedEffect(Unit) {
-                Logger.d("MainActivity -> Starting background initialization")
                 try {
                     withContext(Dispatchers.IO) {
                         appContainer.chatRepository
                         appContainer.lanTransport
                     }
                     isReady = true
-                    Logger.d("MainActivity -> Background initialization COMPLETE")
                 } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)